src/commands/general/profile.ts

```diff
@@ -0,0 +1,27 @@
+import responses from "../../constants/responses";
+import type { Command } from "../../types/Command";
+import fetchUser from "../../util/fetchUser";
```

Comment:
why use `../../util/fetchUser` as a middleman instead of using `node-fetch` directly?